cli/flags.rs

```diff
@@ -145,6 +145,19 @@ To get help on the another subcommands (run in this case):
         .help("Load compiler configuration file")
         .takes_value(true)
         .global(true),
+    ).arg(
+      Arg::with_name("seed")
+        .long("seed")
+        .value_name("NUMBER")
+        .help("Seed Math.random()")
+        .takes_value(true)
+        .validator(|val: String| {
+          match val.parse::<i64>() {
+            Ok(_) => Ok(()),
+            Err(_) => Err("value should be a number".to_string())
```

Comment:
s/value/Seed/

Reply:
Thanks for the comments, I've addressed them so far.